### PR TITLE
Add leap to sle tech preview with yast2 migration

### PR DESCRIPTION
**Pull request by hjluo**
yast-migration-sle a simplified Leap to SLE migration. 
https://jira.suse.com/browse/SLE-24226
To use this migration tool, we need to set LEAP_TECH_PREVIEW_REPO

- Related ticket: https://openqa.nue.suse.com/tests/8625408
- Needles: N/A
- Verification run: 
  https://openqa.nue.suse.com/t8638142
  https://openqa.nue.suse.com/t8638143
  https://openqa.nue.suse.com/t8638144

### CODE REVIEW

File: tests/migration/online_migration/yast2_migration.pm [hunk 1]
```diff
@@ -150,7 +150,9 @@ sub run {
         zypper_call "rs $service{$addon}\t";    # remove service
     }
 
-    script_run("yast2 migration; echo yast2-migration-done-\$? > /dev/$serialdev", 0);
+    my $migration_cmd = 'migration';
+    $migration_cmd = 'migration_sle' if (get_var('LEAP_TECH_PREVIEW'));
```
Comment (lemon-suse) [inline tests/migration/online_migration/yast2_migration.pm:154]: Suggest change to:
$migration_cmd = get_var('LEAP_TECH_PREVIEW') ? 'migration_sle' : 'migration';

File: tests/migration/online_migration/pre_migration.pm [hunk 2]
```diff
@@ -24,6 +24,12 @@ sub check_or_install_packages {
             record_soft_failure('bsc#1197268', 'suseconnect-ng obsoletes zypper-migration-plugin in leap to sle migration');
             zypper_call('rm zypper-migration-plugin');
             zypper_call "in yast2-registration rollback-helper";
+            if (get_var('LEAP_TECH_PREVIEW')) {
```
Comment (lemon-suse) [inline tests/migration/online_migration/pre_migration.pm:27]: Maybe you can user one setting, only the LEAP_TECH_PREVIEW_REPO. WDYT?

File: tests/migration/online_migration/yast2_migration.pm (hunk 1 above)
Comment (hjluo) [reply tests/migration/online_migration/yast2_migration.pm]: changed && thanks

File: tests/migration/online_migration/pre_migration.pm (hunk 2 above)
Comment (hjluo) [reply tests/migration/online_migration/pre_migration.pm]: yes

**Review by tinawang123 — APPROVED** [review]
LGTM

**Review by lemon-suse — APPROVED** [review]

**Comment by lemon-suse** [comment]
Please add the preview link in the description for people to search. Thanks.

**Pull request merged**